docs/usage_guide.rst

```diff
@@ -367,6 +367,15 @@ To stop tracking a particular account scroll down to the accounts tables and cli
 
 If an ethereum account also contains tracked tokens you can click on the arrow under "Actions" in order to expand its view and show the balance breakdown for the account over all assets it holds.
 
+For Bitcoin you can add addresses manually or let rotki discover them using a public key. From this key rotki knows how to generate your addresses and can query the Bitcoin's blockchain generating them until it finds an address without transactions.
```

Comment:
`a public key` -> either say `an xpub` or an `extended public key`

`key rotki knows how to` -> `key rotki can`

`can query the Bitcoin's blockchain generating them until it finds an address without transactions` -> `query the Bitcoin blockchain for each one of them until it finds unused addresses`.